tests/chain_integration/test_exchange_proxy.py

```diff
@@ -1,15 +1,23 @@
 import gevent
 import pytest
 from eth_utils import to_checksum_address
-from ethereum import tester
 
 from relay.blockchain.exchange_proxy import ExchangeProxy
 from relay.constants import NULL_ADDRESS
 from relay.exchange.order import SignableOrder
 
 
+@pytest.fixture
+def accounts(tester, web3):
```

Comment:
We can also remove this here and just alwazs make the make explicit. This is probably better. 
For example
```
maker = tester.a0
taker, *rest = accounts
```

Reply:
or even better probably:
```
maker = order.maker_address
```

Reply:
the accounts fixture here is also something I still wanted to look into. I would have probably introduced a makertaker() fixture that would have returned a tuple.

Reply:
this has been addressed in 8faf063fbf40e703f5e29ba43e729d51b6022d71